storage/src/Database.cpp

```diff
@@ -302,15 +305,16 @@ bool Database::store(const std::string& hash, const std::string& pubKey,
             result = true;
             break;
         } else {
-            LOKI_LOG(error, "Could not execute `store` db statement, ec: {}",
+            LOKI_LOG(critical, "Could not execute `store` db statement, ec: {}",
                      rc);
             break;
         }
     }
 
     rc = sqlite3_reset(stmt);
     if (rc != SQLITE_OK && rc != SQLITE_CONSTRAINT) {
```

Comment:
Unrelated but looking at this code again I think we should probably be setting result = true for SQLITE_CONSTRAINT above as well right?

Reply:
I don't think we should. If we return `true`, that would notify all listeners about a message that they should already have.